src/components/progressCircular/progress-circular.js

```diff
@@ -27,8 +27,10 @@ angular.module('material.components.progressCircular', [
  * indeterminate indicator.
  *
  * @param {string} md-mode Select from one of two modes: **'determinate'** and **'indeterminate'**.<br/>
- * Note: if the `md-mode` value is undefined or not 1 of the two (2) valid modes, then `.ng-hide`
- * will be auto-applied as a style to the component.
+ * <br/>Note: if the `md-mode` value is set as undefined or specified as not 1 of the two (2) valid modes, then `.ng-hide`
+ * will be auto-applied as a style to the component.<br/>
+ * <br/>Note: if not configured, the `md-mode="indeterminate"` will be auto injected as an attribute.
+ * If `value=""` is also specified, however, then `md-mode="determinate"` would be auto-injected instead.
  * @param {number=} value In determinate mode, this number represents the percentage of the
```

Comment:
I thought you could just add newlines in here rather than manually adding `<br/>`.  I could be wrong, though.